components/formats-gpl/src/loci/formats/services/MDBService.java

```diff
@@ -58,6 +58,21 @@
    */
   public Vector<Vector<String[]>> parseDatabase() throws IOException;
 
+  /**
+   * Read named table from a pre-initialized .mdb files.
+   * Each String[] in a table represents a row, and each element of the String[]
+   * represents the value of a particular column within the row.
+   *
+   * The first row in the table contains the names for each column.
+   *
+   * {@link #initialize(String)} must be called before calling parseTable.
+   *
+   * @param name table name
+   * @return table data or null if the named table does not exist
+   * @throws IOException if there is a problem reading the table data
+   */
+  public Vector<String[]> parseTable(String name) throws IOException;
```

Comment:
Since this is an interface we might want to add a default method returning `null` to avoid a breaking change